docs/content/guide/es/10_storages.ngdoc

```diff
@@ -107,15 +107,16 @@ Y listo. Su aplicación ahora usará localStorage para recordar el lenguaje del
     <div ng-controller="Ctrl">
       <p>{{ 'HEADLINE' | translate }}</p>
       <p>{{ 'PARAGRAPH' | translate }}</p>
-
+      
       <p translate>PASSED_AS_TEXT</p>
       <p translate="PASSED_AS_ATTRIBUTE"></p>
       <p translate>{{ 'PASSED_AS_INTERPOLATION' }}</p>
       <p translate="{{ 'PASSED_AS_INTERPOLATION' }}"></p>
-      <p translate="VARIABLE_REPLACEMENT" translate-values="{ name: 'PascalPrecht' }"></p>
-
-      <button ng-click="changeLanguage('de')" translate="BUTTON_LANG_DE"></button>
-      <button ng-click="changeLanguage('en')" translate="BUTTON_LANG_EN"></button>
+      <p translate="VARIABLE_REPLACEMENT" translate-values="{ nombre: 'PascalPrecht'}"></p>
+      <p translate="VARIABLE_REPLACEMENT" translate-value-nombre="PascalPrecht"></p>
```

Comment:
Is this an addition compared with the other language-docs or was it only diffed on an older version? We shouldn't have examples that are not in-sync.